lib/util/git/index.ts

```diff
@@ -342,7 +386,6 @@ export async function syncGit(): Promise<void> {
     }
   }
   if (clone) {
-    await fs.emptyDir(localDir);
```

Comment:
Why?

Reply:
@viceice that is required because `clone` creates files and we should remove them before retrying `clone` again.